src/viewer.js

```diff
@@ -121,12 +110,69 @@ function createViewer(container, data) {
   const renderer = fullScreenRenderer.getRenderer();
   const renderWindow = fullScreenRenderer.getRenderWindow();
 
+  const dataArray = data.image.getPointData().getScalars();
+  if (!dataArray) {
+    window.alert('No data array available in dataset');
+  }
+  const dataRange = dataArray.getRange();
+
+  const lookupTable = vtkColorTransferFunction.newInstance();
+  // TODO: Make colormap selectable
```

Comment:
Do you want me to try to improve the UI a little bit with draggable window and preset selection?

Reply:
Yes, please!

Also, I was trying to figure out a way to limit the renderWindow size in pixels, for our container purposes, so it does not hammer the system and become very slow on high DPI screens. Any ideas?

Reply:
I'll give it a try tomorrow.

Are you saying the slowness depend on the resolution of the 3D window? or the piecewise widget?

Reply:
Thanks!

The 3D window -- fullscreen on a 4k display requests many pixels.

Reply:
I'll see if I can add some options to limit the resolution like we do in ParaViewWeb.